### PR TITLE
Add bracci2019 benchmark (competition)

### PR DESCRIPTION
**Pull request by mike-ferguson**
Adds the Bracci2019 Benchmark for the 2024 competition, as created by: [costantinoai](https://github.com/costantinoai).

This consists of a single new benchmark, `Bracci2019.anteriorVTC-rdm`

### CODE REVIEW

**Review by costantinoai — APPROVED** [review]

File: brainscore_vision/benchmarks/bracci2019/benchmark.py
Comment (mschrimpf) [inline brainscore_vision/benchmarks/bracci2019/benchmark.py:94]: mini convention request for `{dataset}-{metric}`
```suggestion
            identifier=f"Bracci2019.{region}-rdm",
```

File: brainscore_vision/data/bracci2019/requirements.txt
Comment (mschrimpf) [inline brainscore_vision/data/bracci2019/requirements.txt:5]: are these all needed for data *use* or only for data *packaging*? If it's only the later, no need to include the requirements here

**Review by mschrimpf — APPROVED** [review]

File: brainscore_vision/benchmarks/bracci2019/benchmark.py
Comment (costantinoai) [reply brainscore_vision/benchmarks/bracci2019/benchmark.py:94]: @mike-ferguson Feel free to make these suggested changes before merging -- sorry, I do not have write access. I am not sure, but probably a similar change is needed in `brainscore_vision/benchmarks/bracci2019/__init_.py`?

**Comment by mike-ferguson** [comment]
@mschrimpf @costantinoai This benchmark now produces scores and can be merged when tests pass!

**Pull request merged**
